packages/renderless/src/chart-bar/index.ts

```
@@ -134,7 +134,7 @@ const getBarSeries = (args) => {
   innerRows.forEach((row) => metrics.forEach((item) => seriesTemp[item].push(row[item])))
 
   series = Object.keys(seriesTemp).map((item) => {
-    let name = !isNull(labelMap[item]) ? labelMap[item] : item
+    let name = !type.isNull(labelMap[item]) ? labelMap[item] : item
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)